importer/Uploader.py

```diff
@@ -28,17 +28,49 @@ def make_aliases(self):
                 new_aliases.append(new_alias)
         return new_aliases
 
-    def add_labels(self, target_item, labels, log):
+    def add_labels(self, target_item, labels, aliases, log):
+        """
+        Add labels and aliases.
+
+        Normally, if the WD item already has a label in $lang
+        and the data object has another one, the new one
+        will be automatically added as an alias. Otherwise
+        (no existing label), it will be added as a label.
+
+        However, the data model supports an optional [aliases]
+        field as well.
+        Its contents will be added directly as an alias, without checking
+        if it could be a label first.
+        """
         for label in labels:
             target_item.get()
-            name = labels[label]['value']
-            lang = labels[label]['language']
+            label_content = labels[label]['value']
+            language = labels[label]['language']
             self.wdstuff.addLabelOrAlias(
-                lang, name, target_item)
+                language, label_content, target_item)
             if log:
                 t_id = target_item.getID()
-                message = "{} ADDED LABEL {} {}".format(t_id, lang, name)
+                message = "{} ADDED LABEL {} {}".format(
+                    t_id, language, label_content)
                 log.logit(message)
+        if len(aliases) > 0:
```

Comment:
It's always an iterable -- it's [instantiated as an empty dict](https://github.com/Vesihiisi/COH-tools/blob/master/importer/Monument.py#L288), so even if there's no content in it, that empty dict is passed to this stage.

Removed the len test here and also in another place with the same logic.